place_tracker/lib/place_details.dart

```diff
@@ -54,23 +54,23 @@ class PlaceDetailsState extends State<PlaceDetails> {
       children: <Widget>[
```

Comment:
Once that linter is built, I'm going buck wild on these. 😄

Reply:
SGTM =)